### PR TITLE
Use Python 3.7 on Travis

### PR DESCRIPTION
**Pull request by millerdev**
@nickpell

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: python
-python: 2.7
+dist: xenial
```
Comment (nickpell) [inline .travis.yml]: Any reason for xenial instead of bionic?

**Review by nickpell — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (millerdev) [reply .travis.yml]: The [travis documentation](https://docs.travis-ci.com/user/languages/python/#python-37-and-higher) says to do it this way.

Comment (nickpell) [reply .travis.yml]: that's good enough for me

**Comment by emord** [comment]
I think that this added 3-5 minutes on every test job. Its difficult to say exactly how much as we also added the python2 tasks back, but on python2.7 we were consistently getting under 40 min (example: https://app.datadoghq.com/notebook/124244/shard%20investigation%20IIO-335). python 3.7 is almost never < 40 (example https://travis-ci.org/dimagi/commcare-hq/builds/560606096)

**Pull request merged**